ns3_adapter/launch/ns3_adapter.launch

```diff
@@ -21,8 +21,17 @@
   <!-- Directory of vehicle config parameters -->
   <arg name="vehicle_config_dir" default="/opt/carma/vehicle/config" doc="The directory containing vehicle config type parameters"/>
 
+  <!-- Global parameters file path -->
+  <!-- Parameters in this file will override any parameters loaded in their respective packages -->
+  <arg name="global_params_file"
+    default="$(arg vehicle_config_dir)/GlobalParamsOverrideRos1.yaml"
```

Comment:
Typical usage in cdasim-config is that this launch file is directly called. So the input can be provided with different name as well if there are multiple different vehicles, such as:
`roslaunch ns3_adapter ns3_adapter.launch
      global_params_file:='/opt/carma/vehicle/config/GlobalParamsOverrideRos1_1.yaml'
      role_id:='carma_1'
      remap_ns:=/hardware_interface/comms"`

Reply:
However, when the ns3-adapter is converted to ros2. we would start using the ros2 GlobalParamsOverride file, same as all other components in the carma-platform.